nat/src/main/java/org/hyperledger/besu/nat/upnp/BesuUpnpServiceConfiguration.java

```diff
@@ -229,6 +229,11 @@ public Executor getRegistryListenerExecutor() {
     return executorService;
   }
 
+  @Override
+  public Executor getRemoteListenerExecutor() {
+    return executorService;
+  }
+
```

Comment:
is this change related to the gradle upgrade?

Reply:
It's related to the UPNP version upgrade.  They needed a new method implemented as they added a method with a non-default implementation to one of their interfaces (in violation of semver).